modules/nf-core/checkqc/main.nf

```diff
@@ -0,0 +1,48 @@
+process CHECKQC {
+    label 'process_low'
```

Comment:
```suggestion
    label 'process_single'
```
There's no threading, so a single core process should be fine no, or does it need the extra resources?

Reply:
No that should be fine, thank you :+1: